packages/libbullet/build.sh

```diff
@@ -0,0 +1,25 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/bulletphysics/bullet3"
+TERMUX_PKG_DESCRIPTION="SKD for real-time collision detection and multi-physics simulation for robotics, VR, etc"
+TERMUX_PKG_GROUPS="science"
+TERMUX_PKG_LICENSE="ZLIB"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="3.24"
+TERMUX_PKG_SRCURL="https://github.com/bulletphysics/bullet3/archive/refs/tags/$TERMUX_PKG_VERSION.tar.gz"
+TERMUX_PKG_SHA256=6b1e987d6f8156fa8a6468652f4eaad17b3e11252c9870359e5bca693e35780b
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_FORCE_CMAKE=true
```

Comment:
Probably not needed. Also please add `TERMUX_PKG_DEPENDS="libc++"`

Reply:
All C or C++ libs should have a dependency on libc++?

Reply:
C++ usually will depend on libc++_shared.so, can check any lib*.so file or executable using ldd or readelf or objdump